.circleci/config.yml

```diff
@@ -93,37 +93,41 @@ jobs:
       is-canary:
         type: string
         default: ""
-    docker:
-    - image: circleci/android:api-29-node@sha256:71d61d6c21b29948d57120f476a83cc322a280979bce355c5a0ad771293ca380
-    environment:
-      JAVA_OPTS: '-Xms512m -Xmx2g'
-      GRADLE_OPTS: '-Xmx3g -Dorg.gradle.daemon=false -Dorg.gradle.jvmargs="-Xmx2g -XX:+HeapDumpOnOutOfMemoryError"'
+    machine:
+      image: ubuntu-1604:202007-01
```

Comment:
👋 Ceyhun, for documentation purposes, can you elaborate on the particular ubuntu version used here? Thanks!

Reply:
No real reason actually :) other than it being used in the docs here: https://circleci.com/docs/2.0/executor-types/#using-machine 
Let me try the latest version there: `ubuntu-2004:202010-01`